### ISSUE TITLE
BLADE F7 45A AIO

### ISSUE THREAD

**Issue by meor97** [iNavFlight/inav#9977]
## Desired Behavior
New target for Inav

**Comment by sensei-hacker** [comment]
The manufacturer's product page indicates the RUSH_BLADE_F7 target is the correct target for that board.
Are you having some trouble with it?

**Comment by sensei-hacker** [comment]
The best way to get new hardware supported is to ask the manufacturer to please get in touch and provide us the necessary information. The best and normal process is that the manufacturer provides us the information about their design so we can support it.

https://github.com/iNavFlight/inav/blob/master/docs/policies/NEW_HARDWARE_POLICY.md

**Comment by meor97** [comment]
It doesnt work. I flash Inav using blade f7 all the sensors are off

**Comment by sensei-hacker** [comment]
Yes, the manufacturer would need to provide information for THIS board.

**Comment by stewwy** [comment]
Any movement on this?
